llm_handler.py

```
@@ -141,6 +143,14 @@ async def call_llm_for_summary(messages, channel_name, date, hours=24):
         # Join the messages with newlines
         messages_text = "\n".join(formatted_messages_text)
 
+        # Truncate input if it's too long to avoid token limits
+        # Rough estimate: 1 token ≈ 4 characters, leaving room for prompt and response
+        max_input_length = 60000  # ~15k tokens for input, allowing room for system prompt and output
```

Pylint (3.3.7)

[convention] Line too long (101/100)

(C0301)